### ISSUE TITLE
Rev 13:1, 5-6,  16:11, 17:3   blasphemy/insult

### ISSUE THREAD

**Issue by SusanQuigley** [Door43/ulb-en#152]
ULB  Rev 13:1, 5-6,  16:11, 17:3  use "insult" instead of "blaspheme." 
We have a tW page for 'blasphemy" but not for "insult."
Would it be good to use blaspheme and blasphemy instead?
Rev 16:9 has "blasphemed."
Forms of blaspheme are used 25 times in the ULB-NT.

**Comment by TomWarren** [comment]
ULB Rev 13:1 now reads: 
\v 1 Then I saw a beast coming up out of the sea. He had ten horns and seven heads. On his horns were ten crowns, and on each of his heads was a blasphemous name.

ULB Rev 13:5-6 now reads:
\v 5 The beast was given a mouth that could speak proud words and blasphemies . He was permitted to exercise authority for forty-two months.
\v 6 So the beast opened his mouth to speak blasphemies against God: blaspheming his name, the place where he lived, and those who live in heaven.

ULB Rev 16:11 now reads: 
\v 11 They blasphemed the God of heaven because of their pain and sores, and they still refused to repent for what they had done.

ULB Rev 17:3 now reads: 
\v 3 The angel carried me away in the Spirit to a wilderness, and I saw a woman sitting on a scarlet beast that was full of blasphemous names. The beast had seven heads and ten horns.
